.github/workflows/helm-chart-ci.yaml

```diff
@@ -96,7 +96,7 @@ jobs:
       - name: Run chart-testing (lint)
         run: |
           ct lint --debug ${{ github.base_ref != 'release' && '--check-version-increment=false' || '' }} \
-            --target-branch ${{ github.base_ref }}
+            --target-branch ${{ github.base_ref }} --excluded-charts spire-lib
```

Comment:
Lint doesn't support library charts at the moment.

Reply:
No need to exclude as ct lint works on the top level charts.

Reply:
Its something we will need to add once we break out the chart, is related to adding the new chart, and doesn't hurt anything now. I can remove it, but should we just get it out of the way now?

Last I tried, I hit https://github.com/helm/chart-testing/issues/237

Reply:
It may be the charts/common/values.yaml blank file that is allowing it to work. If I tried that, would you prefer having a blank values.yaml over excluding it?

Reply:
Yes that would be better I think